lib/index.js

```diff
@@ -94,6 +94,11 @@ Server.prototype.checkRequest = function(req, fn) {
 Server.prototype.serveClient = function(v){
   if (!arguments.length) return this._serveClient;
   this._serveClient = v;
+
+  if(v && !clientSource) {
+      clientSource = read(require.resolve('socket.io-client/socket.io.js'), 'utf-8');
```

Comment:
Probably it will be better. For me the only reason to mess with this was that require.resolve() is called always but is not emulated by webpack/browserify i.e. it is breaking my application for something that I don't use at all. To be honest I don't see any benefit to serve the client here - may be it is handy for one scenario but not for others. So my goal was to make it more optional and used just on demand without messing too much.